spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jms/JmsProperties.java

```diff
@@ -154,6 +154,11 @@ public void setSessionCacheSize(int sessionCacheSize) {
 		 */
 		private Integer maxConcurrency;
 
+		/**
+		 * Timeout to use for receive calls.
+		 */
+		private Duration receiveTimeout;
```

Comment:
The current default is 1 second so it would be nice to have that documented and tested.